<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>795</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/icon/icons/uo.png</normaloff>:/icon/icons/uo.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>795</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNovo"/>
    <addaction name="actionSave"/>
    <addaction name="actionAbrir"/>
    <addaction name="actionFechar"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuRun">
    <property name="title">
     <string>Run</string>
    </property>
    <addaction name="actionRun_F9"/>
    <addaction name="actionPause_F10"/>
    <addaction name="actionStop_F11"/>
    <addaction name="actionStop_All_F12"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionSkills"/>
   </widget>
   <addaction name="menuArquivo"/>
   <addaction name="menuRun"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNovo"/>
   <addaction name="actionSave"/>
   <addaction name="actionAbrir"/>
   <addaction name="actionReopen"/>
   <addaction name="separator"/>
   <addaction name="actionRun_F9"/>
   <addaction name="actionPause_F10"/>
   <addaction name="actionStep"/>
   <addaction name="actionStop_F11"/>
   <addaction name="actionStop_All_F12"/>
   <addaction name="separator"/>
   <addaction name="actionSwap"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget_6">
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>315</height>
    </size>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_6">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QTreeView" name="treeView">
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="rootIsDecorated">
        <bool>true</bool>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="headerCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="headerHighlightSections">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionNovo">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/icons/new_icon.png</normaloff>:/icon/icons/new_icon.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionAbrir">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/icons/open_icon.png</normaloff>:/icon/icons/open_icon.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionFechar">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionRun_F9">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/icons/play_icon.png</normaloff>:/icon/icons/play_icon.png</iconset>
   </property>
   <property name="text">
    <string>Run (F9)</string>
   </property>
   <property name="shortcut">
    <string>F9</string>
   </property>
  </action>
  <action name="actionPause_F10">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/icons/pause_icon.png</normaloff>:/icon/icons/pause_icon.png</iconset>
   </property>
   <property name="text">
    <string>Pause(F10)</string>
   </property>
   <property name="shortcut">
    <string>F10</string>
   </property>
  </action>
  <action name="actionStop_F11">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/icons/stop_icon.png</normaloff>:/icon/icons/stop_icon.png</iconset>
   </property>
   <property name="text">
    <string>Stop(F11)</string>
   </property>
   <property name="shortcut">
    <string>F11</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionStop_All_F12">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/icons/stopall_icon.png</normaloff>:/icon/icons/stopall_icon.png</iconset>
   </property>
   <property name="text">
    <string>Stop All (F12)</string>
   </property>
   <property name="shortcut">
    <string>F12</string>
   </property>
  </action>
  <action name="actionReopen">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/icons/reopen_icon.png</normaloff>:/icon/icons/reopen_icon.png</iconset>
   </property>
   <property name="text">
    <string>Reopen</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/icons/save_icon.png</normaloff>:/icon/icons/save_icon.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSwap">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/icons/swap_icon.png</normaloff>:/icon/icons/swap_icon.png</iconset>
   </property>
   <property name="text">
    <string>Swap</string>
   </property>
   <property name="toolTip">
    <string>Swap UO client</string>
   </property>
  </action>
  <action name="actionSkills">
   <property name="text">
    <string>Skills</string>
   </property>
  </action>
  <action name="actionStep">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/icons/step_icon.png</normaloff>:/icon/icons/step_icon.png</iconset>
   </property>
   <property name="text">
    <string>Step</string>
   </property>
   <property name="toolTip">
    <string>Step Line</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
